### DISCUSSION TITLE
Hanging - how to debug?

### DISCUSSION THREAD

**Discussion by mramendi** [pulsar-edit/.github#179] [Core Application & Packages]
So Pulsar just hung for me. After some time it showed a window informing me that the editor is not recponsing and offering a forced close, which I did.

Not much is seen in the console output:

````
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: worker-manager.js:294:26
Remote event names: crashed, destroyed
````

If this happens again, how do I collect useful information so an issue can be raised?

**Comment by confused-Techie** [comment]
I'd really hope the console would provide what we need to figure out what happened. A performance report could also help, but I'm assuming there is no way to get that once it becomes unresponsive. So it's a bit of a hard one.

The best thing I can think of beyond that would be to try and determine what you were doing right before it hung, such as if you were editing a file, what grammar was being used, and what packages might have been active. Since really finding a way to reliably reproduce this issue would be our best bet into determining where it's going wrong. So any information that can be used to reproduce this issue would likely be the most helpful for now to figure out what's causing it.

But otherwise I do really appreciate you reporting this one
